partials/shop-product-list.htm

```diff
@@ -1,5 +1,5 @@
 {{ open_form() }}
-    {% paginate products by 48 %}
+    {% paginate products by 20 %}
```

Comment:
@robotpony yah, they totally should. We can discuss whether to add them in now or later.